selfdrive/car/volkswagen/carcontroller.py

```diff
@@ -39,7 +39,7 @@ def update(self, enabled, CS, frame, ext_bus, actuators, visual_alert, left_lane
       # torque value. Do that anytime we happen to have 0 torque, or failing that,
       # when exceeding ~1/3 the 360 second timer.
 
-      if enabled and not (CS.out.standstill or CS.out.steerError or CS.out.steerWarning):
+      if enabled and CS.out.vEgo > CS.CP.minSteerSpeed and not (CS.out.standstill or CS.out.steerError or CS.out.steerWarning):
```

Comment:
`CS.out.vEgo > CS.CP.minSteerSpeed` where minSteerSpeed == default of 0 is almost the same thing as `CS.out.standstill`, but I'd like to keep both conditions.

In the early days of the port, I used `vEgo < 0.01` as my standstill condition and had problems with EPS faulting coming to a standstill until I raised it to `vEgo < 0.1` (since replaced with the EPS hydraulic hold flag).